pkgs/python-packages/default.nix

```diff
@@ -5,7 +5,7 @@ let
   # be supported through the NixOS 23.05 release cycle.`
   # TODO-EXTERNAL: consider removing when
   # https://github.com/Simplexum/python-bitcointx/issues/76 and
-  # https://github.com/JoinMarket-Org/joinmarket-clientserver#1451 are resolved.
+  # https://github.com/JoinMarket-Org/joinmarket-clientserver/issues/1451 are resolved.
```

Comment:
How about
```
consider removing when a fix to
https://github.com/JoinMarket-Org/joinmarket-clientserver/issues/1613
has been released.
```

Reply:
Resolved by the `python-bitcointx` update (see post below).

Reply:
This comment block doesn't seem to be updated?